main.go

```diff
@@ -20,6 +21,9 @@ import (
 // set dynamically on build time. See Makefile for more info
 var version string
 
+// set dynamically on build time. If this value is "true", it will skip unnecessary external calls
+var runningTests string
```

Comment:
It was a bit tricky to run the full test suite without having a flag to skip the Github check. It doesn't feel the best but I'm open to suggestions if there's a better way!